client/blocks/upload-image/style.scss

```diff
@@ -56,3 +56,82 @@
 	top: 50%;
 	transform: translate( -50%, -50% );
 }
+
+.upload-image__image-container {
+	width: 200px;
+	height: 200px;
+
+	&.is-uploaded, &.is-uploading {
+		border: 1px solid lighten( $gray, 10% );
+		background-color: lighten( $gray, 30% );
+	}
+}
+
+.upload-image__uploading-container, .upload-image__uploading-done-container {
+	height: 100%;
+	text-align: center;
+}
+
+.upload-image__uploading-done-container {
+	position: relative;
+}
+
+.button.is-compact.upload-image__uploaded-image-remove {
+	position: absolute;
+		top: 0;
+		right: 0;
+	width: 24px;
+	height: 24px;
+	transform: translate(50%, -50%);
+	border: 1px solid lighten( $gray, 10% );
+	border-radius: 50%;
+	background-color: $gray-light;
+	cursor: pointer;
+	z-index: z-index( '.dialog__backdrop', '.upload-image__uploaded-image-remove' );
+}
+
+.button.is-compact .gridicon.upload-image__uploaded-image-remove-icon {
+	position: absolute;
+		top: 50%;
+		left: 50%;
+	transform: translate(-50%, -50%);
```

Comment:
No worries, it's a perfectly valid solution, just not my personal favorite :)
![c_dpm0xw0aaoq1g](https://user-images.githubusercontent.com/664258/28714686-c4071218-7394-11e7-8641-5797085d2040.jpg)